website/docs/docs/concepts/00-resource-group-definitions.md

```diff
@@ -137,7 +137,10 @@ correctness and set up the necessary components:
    - Checks that referenced values exist and are of the correct type
    - Confirms resource dependencies form a valid Directed Acyclic Graph(DAG)
      without cycles
-   - Validates all CEL expressions in status fields and conditions
+   - Validates all CEL expressions in status fields and conditions using CEL's native type system
+     - Validates field references exist in the actual resource schemas
+     - Ensures expressions return types compatible with their target fields
```

Comment:
Not sure if it's worth calling out here, but presumably this includes validating that the functions you're calling exist so it's not only that expressions return compatible types, it validates the expression "correctness" (not sure how to call it) as well, right?

Reply:
Good callout will add that.